test/functional/apps/dashboard/group3/panel_context_menu.ts

```diff
@@ -79,6 +78,7 @@ export default function ({ getService, getPageObjects }: FtrProviderContext) {
 
     describe('visualization object edit menu', () => {
       it('opens a visualization when edit link is clicked', async () => {
+        await dashboardPanelActions.openContextMenu();
         await dashboardPanelActions.clickEdit();
```

Comment:
`clickEdit` expects the context menu to be open. so call the `openContextMenu` method first.